faces-tck/run-tck.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
-if [ JAVA_HOME = "" ] ; then
+#if [ JAVA_HOME = "" ] ; then
     export JAVA_HOME=/usr/lib/jvm/java-21-openjdk-amd64
-fi
+#fi
+
+#echo "Downloading Faces TCK"
+#cd ..
+#mvn clean install -pl .,:tck-download,:jakarta-faces-tck
+#cd faces-tck
+
+#echo "Building old tck, expected time 3-7 minutes"
+#cd target/faces-tck-4.1.1/tck/old-tck
+#mvn clean install -Dtck.mode=platform -Dmojarra.noupdate=true -Dpayara.version=7.2024.1.Alpha2 -Dfaces.version=4.1.1 -pl :old-faces-tck-parent,:old-tck-build | tee ../../../old-tck.log
+#cd ..
 
-echo "Downloading Faces TCK"
-cd ..
-mvn clean install -pl .,:tck-download,:jakarta-faces-tck
-cd faces-tck
```

Comment:
Did you intend to commit these changes?

This is a script which just runs `mvn clean verify` and does nothing else.
If this is intentional I'd just delete the file as it's pointless.
If anything it's worse than pointless, as it will overwrite _JAVA_HOME_ with an environment specific value

Reply:
I'll move the notes to README and delete the script